tests/datasets/configuration/test_storage_loader.py

```diff
@@ -112,6 +113,15 @@ def _deep_compare_storages(old: Storage, new: Storage) -> None:
             new.get_table_writer().get_stream_loader(),
         )
 
+    if isinstance(old, CdcStorage) or isinstance(new, CdcStorage):
```

Comment:
Is this code actually running on anything? We don't have any CDC storages in config to compare yet right?

Reply:
This is for #3549, thought I'd add the support in this PR so the other PR is cleaner